.gitignore

```diff
@@ -14,3 +15,5 @@ dist/
 
 *.pyc
 *.sublime-workspace
+st_package_reviewer/_version.py
+uv.lock
```

Comment:
Please commit the uv lock file for reproducible environments (locally and in CI).

Reply:
And then always forgot to change that....